### DISCUSSION TITLE
How to change the style of a button?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2055] [Help]
Hello,

I can see on the button that I can chqnge the preset style of a button with primary for example. But it is not changing my style.

Is it normal?

**Comment by GuilhermeHirsch** [comment]
I've tested it here, and I've noticed this same behavior only with auto-generated forms.
In user-created buttons, I can change the button style.
Is that what happens to you?

**Reply by GuilhermeHirsch** [reply]
If so, just delete that one and create one manually with the same actions.

**Comment by oswaldB** [comment]
I will try it.

And is it possible to modify the style anyhow without using the CSS area?

**Reply by GuilhermeHirsch** [reply]
Yes, through the button type
